examples/ffi/crashtracking.c

```diff
@@ -18,21 +18,29 @@ int main(int argc, char **argv) {
     return -1;
   }
 
+  ddog_prof_CrashtrackerReceiverConfig receiver_config = {
+      .args = {},
+      .env = {},
+      .path_to_receiver_binary = DDOG_CHARSLICE_C("FIXME - point me to receiver binary path"),
```

Comment:
Will this be ironed out?  If not, suggest changing the string literal to explicitly tell the user that they are expected to adapt this to their build process.

Reply:
done